tomster-player/config/fixtures.json

```diff
@@ -0,0 +1,320 @@
+      "mp3Url": "https://p.scdn.co/mp3-preview/73cdb98b6f4157d0f0ae1950366b022fcfa9a391",
+      "album": "3"
+    },
+    {
+      "id": "23",
+      "title": "Heartbreak Hotel",
+      "duration": 30000,
+      "mp3Url": "https://p.scdn.co/mp3-preview/4daf3565cae4f18ef9c9700ad9c162596e8eec71",
+      "album": "3"
+    },
+    {
+      "id": "24",
+      "title": "My Love Is Your Love",
+      "duration": 30000,
+      "mp3Url": "https://p.scdn.co/mp3-preview/9a265fe7b4388ee91fc80a122c5ff685259a678d",
+      "album": "3"
+    },
+    {
+      "id": "25",
+      "title": "When You Believe",
+      "duration": 30000,
+      "mp3Url": "https://p.scdn.co/mp3-preview/6bddbd827f3ae93db987505fd9c0da210a980e6e",
+      "album": "3"
+    },
+    {
+      "id": "26",
+      "title": "If I Told You That",
+      "duration": 30000,
+      "mp3Url": "https://p.scdn.co/mp3-preview/d2e60e8f12e7ac462090f41d64ddaa5ae40b51e6",
+      "album": "3"
+    },
+    {
+      "id": "27",
+      "title": "In My Business",
+      "duration": 30000,
+      "mp3Url": "https://p.scdn.co/mp3-preview/43888eaed49ad7fb8d81cded083d7b62589a5cf3",
+      "album": "3"
+    },
+    {
+      "id": "28",
+      "title": "I Learned from the Best",
+      "duration": 30000,
+      "mp3Url": "https://p.scdn.co/mp3-preview/e0f61b4d8709e6d9ba7e65b117ed036bd04de16b",
+      "album": "3"
+    },
+    {
+      "id": "29",
+      "title": "Oh Yes",
+      "duration": 30000,
+      "mp3Url": "https://p.scdn.co/mp3-preview/56b2ed7e5740656a430a138ae6d07ad69cc01d4a",
+      "album": "3"
+    },
+    {
+      "id": "30",
+      "title": "Get It Back",
+      "duration": 30000,
+      "mp3Url": "https://p.scdn.co/mp3-preview/5b5a25efb1c9a967a4174679d35cefc1b0a9d59f",
+      "album": "3"
+    },
+    {
+      "id": "31",
+      "title": "Until You Come Back",
+      "duration": 30000,
+      "mp3Url": "https://p.scdn.co/mp3-preview/ccfe060756207827ca2a94273a295f39ec319332",
+      "album": "3"
+    },
+    {
+      "id": "32",
+      "title": "I Bow Out",
+      "duration": 30000,
+      "mp3Url": "https://p.scdn.co/mp3-preview/eadbf7e5b6573e7bd9e857960b4f565ca158954a",
+      "album": "3"
+    },
+    {
+      "id": "33",
+      "title": "You'll Never Stand Alone",
+      "duration": 30000,
+      "mp3Url": "https://p.scdn.co/mp3-preview/a20cd3990cdc5e10d7a8b487a103aa76c60a717a",
+      "album": "3"
+    },
+    {
+      "id": "34",
+      "title": "I Was Made To Love Him",
+      "duration": 30000,
+      "mp3Url": "https://p.scdn.co/mp3-preview/decd5000c37284d7718d8e504013b035abfcc367",
+      "album": "3"
+    }
+  ],
+  "comment": [
+    {
+      "id": "1",
+      "rating": 1,
+      "text": "Not actually my favorite…",
+      "createdAt": "2016-01-10T11:34:00.000Z",
+      "album": "2"
+    },
+    {
+      "id": "2",
+      "rating": 5,
+      "text": "Such a great album - a total christmas classic for me!",
+      "createdAt": "2015-12-24T17:05:00.000Z",
+      "album": "2"
+    }
+  ]
+}
```

Comment:
We should allow to split the fixtures into separate files - I'd say one per record type.